recipes/newrelic/infrastructure/ohi/apache/debian.yml

```diff
@@ -61,8 +61,8 @@ install:
     assert_pre_req:
       cmds:
         - |
-          SERVICE_EXIST=$(sudo systemctl status newrelic-infra.service | grep "Active" | wc -l)
-          if [ $SERVICE_EXIST -eq 0 ]; then
+          systemctl is-active --quiet newrelic-infra
```

Comment:
I assume this was failing before the grep "Active" was not latching correctly for Ubuntu based host.
If that's the case, we'll want to have this updated on all the Debian OHI recipes.